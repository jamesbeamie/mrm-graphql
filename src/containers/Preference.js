import React, { Fragment } from 'react';
import TopMenu from '../components/navbars/TopMenu';
import NotificationSettingsList from '../components/preference/notificationsSettings/NotificationSettingsList';
import '../assets/styles/preference.scss';
import ROUTES from '../utils/routes';
import BackArrow from '../../src/components/commons/BackArrow';

/**
 * render all preference settings
 */
const Preference = () => (
  <Fragment>
    <TopMenu />
    <div className="preference-container">
      <BackArrow redirectUri={ROUTES.analytics} />
      <NotificationSettingsList />
    </div>
  </Fragment>
);

export default Preference;
