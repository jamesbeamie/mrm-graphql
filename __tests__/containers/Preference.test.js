import React from 'react';
import { shallow } from 'enzyme';

import Preference from '../../src/containers/Preference';

describe('Preference Component', () => {
  const wrapper = shallow(<Preference />);

  it('should render properly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
