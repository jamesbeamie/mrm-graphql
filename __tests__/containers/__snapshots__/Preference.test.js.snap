// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Preference Component should render properly 1`] = `
<Fragment>
  <withRouter(TopMenuComponent) />
  <div
    className="preference-container"
  >
    <BackArrow
      redirectUri="/analytics"
    />
    <NotificationSettingsList />
  </div>
</Fragment>
`;
